javascript/modules/client/README.md

```diff
@@ -69,7 +69,7 @@ const creationParams: ICreateParams = {
     name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
     metadata: "<ipfs-http-uri>",
   },
-  votingConfig: {
+  : {
     minSupport: Math.floor(Math.random() * 100) + 1,
     minParticipation: Math.floor(Math.random() * 100) + 1,
     minDuration: Math.floor(Math.random() * 9999) + 1,
```

Comment:
This needs to be changed accordingly to the documentation, see below

Reply:
Same